File: C/EX15.c
/*Calcula o valor do imc*/
#include<stdlib.h>
#include<stdio.h>

int main(int argc, char const *argv[])
{
    float imc, peso, altura;

    printf("Informe sua: \nAltura:");
    scanf("%f", &altura);

    printf("Peso: ");
    scanf("%f", &peso);

    imc = peso / (altura * altura);

    if (imc < 20)
    {
        printf("Abaixo do peso.");
    }

    else if (imc >= 25)
    {
        printf("Acima do peso.");
    }

    else
    {
        printf("Peso ideal.");
    }
       
    return 0;
}

File: Python/EX15.py
#cáculo do imc

peso = float(input("Peso: "))
altura = float(input("Altura: "))

imc = peso / (altura**2)

if imc < 20:
    print("Abaixo do peso.")
elif  imc >= 25:
    print("Acima do peso.")
else:
    print("peso ideal")
